ui/components/app/name/index.scss

```diff
@@ -9,14 +9,30 @@
 
   &__missing {
     background-color: var(--color-error-muted);
+
+    .name__value {
```

Comment:
We could use SCSS variables to avoid the duplicate `.name` but that would probably result in more complexity rather than less 😄

Reply:
Yes, because we're already nesting inside $__missing etc., I couldn't find a readable way of avoiding the duplicate .name.